pkg/util/spanlogger/spanlogger.go

```diff
@@ -9,11 +9,16 @@ import (
 	"github.com/opentracing/opentracing-go/ext"
 	otlog "github.com/opentracing/opentracing-go/log"
 
+	"github.com/cortexproject/cortex/pkg/tenant"
 	util_log "github.com/cortexproject/cortex/pkg/util/log"
 )
 
 type loggerCtxMarker struct{}
 
+const (
+	TenantIDTagName = "tenant_id"
```

Comment:
should this maybe be plural? since you're setting a `[]string` value
```suggestion
	TenantIDTagName = "tenant_ids"
```

Reply:
done